src/Symfony/Bundle/FrameworkBundle/Test/BrowserKitAssertionsTrait.php

```diff
@@ -124,6 +126,33 @@ public static function assertRouteSame($expectedRoute, array $parameters = [], s
         self::assertThat(self::getRequest(), $constraint, $message);
     }
 
+    public static function assertThatForResponse(Constraint $constraint, string $message = ''): void
+    {
+        try {
+            self::assertThat(self::getResponse(), $constraint, $message);
+        } catch (ExpectationFailedException $exception) {
+            if (($serverExceptionMessage = self::getResponse()->headers->get('X-Debug-Exception'))
+                && ($serverExceptionFile = self::getResponse()->headers->get('X-Debug-Exception-File'))) {
+
+                $serverExceptionFile = explode(':', $serverExceptionFile);
```

Comment:
this should split at the last occurrence of a `:` - but here this is the first

Reply:
oh, nevermind: since we expected the string to be urlencoded, there can be no previous `:`